docs/user_guide/model-registration.rst

```diff
@@ -82,7 +88,10 @@ the processing output constitutes a complete model registration.
 Continuous Integration
 ----------------------
 
-In most cases, a model registration is submitted as a pull request to a project repository hosted on GitHub. As part
-of this, :func:`assert_valid_structure` (details can be found here: :ref:`cli`) is run
-automatically to ensure that the model registration is valid. Any regions that are, for
-example mentioned in a mapping but not defined will raise an error.
+When registering a model using a pull request to a project repository hosted on GitHub,
```

Comment:
This sentence does not make sense to me.

Reply:
Fixed and shortened.